gateway/webhook/setup.go

```diff
@@ -59,6 +62,7 @@ func Setup(gwHandle Gateway, stat stats.Stats, opts ...batchTransformerOption) *
 		webhook.config.sourceListForParsingParams[i] = strings.ToLower(s)
 	}
 
+	webhook.transformerFeaturesService = transformerFeaturesService
```

Comment:
We need to save the transformerFeaturesService handle because, we need to wait till transformer features are fetched. This check is performed in `batchTransformLoop`.

Reply:
I proposed an alternative approach before that didn't require storing it in a struct field